### PR TITLE
[browser] easy prep for Firefox

### PR DESCRIPTION
**Pull request by pavelsavara**

### CODE REVIEW

**Review by lewing — APPROVED** [review]

File: src/mono/browser/test-main.js [hunk 1]
```diff
@@ -312,6 +321,12 @@ async function run() {
         App.runtime = await dotnet.create();
         App.runArgs = runArgs
 
+        if (ENVIRONMENT_IS_WEB && isFirefox) {
+            console.log("Application arguments: " + runArgs.applicationArguments.join(' '));
```
Comment (ilonatommy) [inline src/mono/browser/test-main.js:325]: It's a duplication of code from line 308 (for Firefox).
```suggestion
```

File: src/mono/browser/test-main.js [hunk 2]
```diff
@@ -312,6 +321,12 @@ async function run() {
         App.runtime = await dotnet.create();
         App.runArgs = runArgs
 
+        if (ENVIRONMENT_IS_WEB && isFirefox) {
```
Comment (ilonatommy) [inline src/mono/browser/test-main.js:324]: Why this is in `async function run()` when for chrome we do out of any function (line 60)? When logically it does not change much, it's confusing.

**Review by ilonatommy — APPROVED** [review]

File: src/mono/browser/test-main.js (hunk 1 above)
Comment (pavelsavara) [reply src/mono/browser/test-main.js:325]: In firefox we would not see any `console.log` until the console proxy was installed.

File: src/mono/browser/test-main.js (hunk 2 above)
Comment (pavelsavara) [reply src/mono/browser/test-main.js:324]: In chrome it's best to print it as soon as possible, and chrome driver would give it to us. In firefox it doesn't work.

**Pull request merged**